msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -65,8 +65,11 @@ The following table provides resource-specific permissions for a chat:
 
 For more details, see [chat resource-specific consent permissions](/graph/permissions-reference#chat-resource-specific-consent-permissions).
 
+### RSC Application permission
+
 > [!NOTE]
-> Resource-specific permissions are only available to Teams apps installed on the Teams client and are currently not part of the Azure Active Directory (AAD) portal.
+> * RSC Application permission includes all the application permissions of older version [Resource-specific permissions for a team](#resource-specific-permissions-for-a-team), [Resource-specific permissions for a chat](#resource-specific-permissions-for-a-chat).
```

Comment:
The sentence doesnt sound right. Please rephrase.

Reply:
Rephrased.